lib/portage/dbapi/bintree.py

```diff
@@ -964,6 +964,11 @@ def _populate_remote(self, getbinpkg_refresh=True):
 						download_timestamp + ttl > time.time():
 						raise UseCachedCopyOfRemoteIndex()
 
+				# Set proxy settings for _urlopen -> urllib_request
```

Comment:
@mgorny, @zmedico : I've also solved an issue with BINPKG host set and proxy set.

Reply:
I suppose it would be cleaner to pass it through `_urlopen` explicitly.

Reply:
I pass proxy settings directly as parameter and parsed for the `ProxyHandler`.